tests/wsl/install/ms_win_installation.pm

```diff
@@ -14,14 +14,32 @@ use warnings;
 
 use testapi;
 
+sub prepare_win11 {
+    # Change some regedit values to skip system requirements check
+    send_key 'shift-f10';
+    assert_screen 'windows-install-cmd';
+    type_string "reg add HKEY_LOCAL_MACHINE\\SYSTEM\\Setup /v LabConfig\n";
+    type_string "reg add HKEY_LOCAL_MACHINE\\SYSTEM\\Setup\\LabConfig /v BypassTPMCheck /t REG_DWORD /d 1\n";
+    type_string "reg add HKEY_LOCAL_MACHINE\\SYSTEM\\Setup\\LabConfig /v BypassRAMCheck /t REG_DWORD /d 1\n";
+    type_string "reg add HKEY_LOCAL_MACHINE\\SYSTEM\\Setup\\LabConfig /v BypassSecureBootCheck /t REG_DWORD /d 1\n";
```

Comment:
Cannot you use `powershell` here over `cmd`?
Does the `secureboot` check somehow affects non uefi jobs? That might be actually a good thing to have, as WSL should run without any problems on UEFI and legacy machines.

Reply:
This is the command shell called with `Shift+F10` from the installation screen, not sure if PowerShell is available at that moment.
About the non-UEFI, I'm quite sure that Win11 does not support non-UEFI boot, but I'll have to check...